src/main/java/com/exasol/projectkeeper/validators/changesfile/dependencies/DependencyChangeReportRenderer.java

```diff
@@ -19,10 +19,10 @@
     public List<String> render(final DependencyChangeReport report) {
         final List<String> lines = new ArrayList<>();
         lines.add(ChangesFile.DEPENDENCY_UPDATES_HEADING);
-        addDependencyChanges("### Compile Dependency Updates", report.getCompileDependencyChanges(), lines);
-        addDependencyChanges("### Runtime Dependency Updates", report.getRuntimeDependencyChanges(), lines);
-        addDependencyChanges("### Test Dependency Updates", report.getTestDependencyChanges(), lines);
-        addDependencyChanges("### Plugin Dependency Updates", report.getPluginDependencyChanges(), lines);
+        addDependencyChanges("### Compile ProjectDependency Updates", report.getCompileDependencyChanges(), lines);
```

Comment:
I think this refactoring was not intended? :)

Reply:
ups...